docs/use_cases/otel/insights.md

```diff
@@ -0,0 +1,127 @@
+# Insights
```

Comment:
The insights are what the human has. I would call this section "data processing" or "extracting high-level metrics"